doctr/models/predictor/tensorflow.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         self.reco_predictor = reco_predictor
         self.doc_builder = DocumentBuilder(export_as_straight_boxes=export_as_straight_boxes)
         self.assume_straight_pages = assume_straight_pages
+        self._children_names = ['det_predictor', 'reco_predictor']
```

Comment:
Indeed, would you mind moving this to a class attribute rather than instance attribute? (i.e. at line 29 similarly to other TF components)